test/fake_slack.ml

```diff
@@ -0,0 +1,121 @@
+(* A (currently pretty useless) fake slack implementation to run tests
+   against. *)
+open Lwt
+open Cohttp
+open Cohttp_lwt_unix
+
+
+let valid_token = "xoxp-testtoken" (* The only "valid" token we accept. *)
```

Comment:
Can you put this comment above the line? This prevents lines from getting too unwieldy long and ugly.